tasks/kubelet-setup.yml

```diff
@@ -1,35 +1,42 @@
 ---
-- name: Check for existence of kubelet environment file.
+
+# ---- DEPRECATED ----------------
```

Comment:
I don't think we need to add a notice here as well; we already have it in the vars and README, and in the description of this task.

Reply:
ok. I first had all the notices in the files, when I finally added them to the README